modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -1734,15 +1734,17 @@ private CompletableFuture<?> processMultiEntryAction(ReadWriteMultiRowReplicaReq
 
                 return allOf(deleteExactLockFuts).thenCompose(ignore -> {
                     Map<UUID, BinaryRowMessage> rowIdsToDelete = new HashMap<>();
-                    Collection<BinaryRow> result = new ArrayList<>();
+                    Collection<Boolean> result = new ArrayList<>();
```

Comment:
`java.util.BitSet` will be more effective.

Reply:
This appoach is used before for the delete all operation. I just eaxtend it to deleteAllExct and insertAll operations.